src/modal/modal.spec.ts

```diff
@@ -615,7 +615,26 @@ describe('ngb-modal', () => {
       fixture.detectChanges();
       expect(fixture.nativeElement).not.toHaveModal();
     });
+  });
+
+  describe('accessibility', () => {
+    function testAttributes(modalOptions, expectedAttributes) {
+      const modalInstance = fixture.componentInstance.open('foo', modalOptions);
+      fixture.detectChanges();
 
+      const modalElement = <HTMLElement>document.querySelector('ngb-modal-window');
+      expectedAttributes.forEach(([name, value]) => expect(modalElement.getAttribute(name)).toBe(value));
+
+      modalInstance.close('some result');
+      fixture.detectChanges();
+      expect(fixture.nativeElement).not.toHaveModal();
+    }
+
+    it('should support aria-labelledby', () => {
+      const id = 'aria-labelledby-id';
+
+      testAttributes({ariaLabelledBy: id}, [['aria-labelledby', id]]);
```

Comment:
Can't we simply write `expect(modalElement.getAttribute('ariaLabelledBy')).toBe('aria-labelled-by-id')`?
It's the only place `testAttributes` is used

Reply:
Sure, it's a leftover from previous version, where it was used multiple times. I will remove the generic function.